tests/ha/single_pod_restart/test_data_pod_restart.py

```diff
@@ -271,96 +269,95 @@ def test_write_after_pod_restart(self):
-
-        LOGGER.info("Step 8: Perform READs and verify DI on the written data")
+        if CMN_CFG["dtm0_disabled"]:
+            LOGGER.info("Step 5: Perform READs and verify DI on the written data in degraded "
+                        "cluster with new buckets and objects.")
+            resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
+                                                        log_prefix=self.test_prefix_deg,
+                                                        skipwrite=True, skipcleanup=True,
+                                                        setup_s3bench=False)
+            assert_utils.assert_true(resp[0], resp[1])
+            LOGGER.info("Step 5: Successfully run READ/Verify on data written in degraded cluster "
+                        "with new buckets and objects.")
+        LOGGER.info("Step 5: Perform READs and verify DI on the written data with buckets created "
+                    "in healthy cluster.")
         resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
-                                                    log_prefix=self.test_prefix, skipwrite=True)
+                                                    log_prefix=self.test_prefix, skipwrite=True,
+                                                    skipcleanup=True, setup_s3bench=False)
         assert_utils.assert_true(resp[0], resp[1])
-        LOGGER.info("Step 8: Performed READs and verified DI on the written data")
+        LOGGER.info("Step 5: Successfully run READ/Verify on data written with buckets created "
+                    "in healthy cluster")
 
-        LOGGER.info("Step 9: Perform WRITEs-READs-Verify with variable object sizes. (0B - 512MB("
-                    "VM)/5GB(HW))")
-        users = self.mgnt_ops.create_account_users(nusers=1)
-        self.test_prefix = 'test-34074-1'
-        self.s3_clean.update(users)
+        LOGGER.info("Step 6: Perform WRITEs-READs-Verify with variable object sizes.")
+        if CMN_CFG["dtm0_disabled"]:
+            LOGGER.info("Create new IAM user and buckets.")
+            users = self.mgnt_ops.create_account_users(nusers=1)
+            self.test_prefix = 'test-34074-restart'
+            self.s3_clean.update(users)
         resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
                                                     log_prefix=self.test_prefix,
-                                                    skipcleanup=True)
+                                                    skipcleanup=True, setup_s3bench=False)
         assert_utils.assert_true(resp[0], resp[1])
-        LOGGER.info("Step 9: Performed WRITEs-READs-Verify with variable sizes objects.")
+        LOGGER.info("Step 6: Performed WRITEs-READs-Verify with variable sizes objects.")
 
         LOGGER.info("ENDED: Test to verify WRITEs after data pod restart.")
```

Comment:
Should we try writing again on bucket created in healthy cluster even when dtm is disabled?

Reply:
So running IOs on already created buckets in healthy cluster is already taken care in step 5, this is additional step to just try creating new buckets after restart, modified this step accordingly.